models/public/hbonet-0.25/hbonet-0.25.md

```diff
@@ -0,0 +1,63 @@
+# hbonet-0.25
+
+## Use Case and High-Level Description
+
+The `hbonet-0.25` model is one of the classification models from https://github.com/d-li14/HBONet with `width_mult=0.25`
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 0.037         |
+| MParams           |               |
+| Source framework  | PyTorch\*     |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 57.30          |                 |
```

Comment:
If you provide accuracy in such way (orig vs converted), you should also provide accuracy of converted model. So, for `original` we use values from paper or obtained from original framework, for `converted` - values for IR from AC.

As option, you could remove one column.